### ISSUE TITLE
Problem with tracking time

### ISSUE THREAD

**Issue by jokaacake** [Staartvin/Autorank-2#483]
Hi there!
The plugin isn't keeping track of the time on the server, I mean, the activity files are registering just fine but when I do /ar time it says I got 0 minutes played. I already done a legit path and that's not the issue.
Possible problems:
- Creative gamemode and OP;
- Using MySQL;
- Runing an offline server;
Hope someone can help me.
Regards

EDIT: I'm using the latest version (v4.0.5).

**Comment by Staartvin** [comment]
Can you turn on debug in the config, restart your server, let it run for a while (few hours) with players on and then report back with the server.log?

**Comment by jokaacake** [comment]
Thank you for your fast answer. I let it run for a while, hope it's enough. Otherwise I'll do it again.

Files:
- Server log: http://pastebin.com/hfuK1mNW
- Settings: http://pastebin.com/tUsqRYpF
- Paths: http://pastebin.com/meJNXPpS
- InternalProps: http://pastebin.com/V2CPkEd6

**Comment by Staartvin** [comment]
Can you try turning 'use partial completion' to true?

**Comment by jokaacake** [comment]
I did it and the problem persists :/
The server.log doesn't show anything different too.

**Comment by Staartvin** [comment]
How do you check your time? And how often?

**Comment by jokaacake** [comment]
I use /ar time to check my time and I'm using it a bit too often when I'm online and not afk since I'm always expecting it to work properly 😃 
Btw, I noticed if I execute /ar activity <nick> <time> command it shows the time correctly, it's working, unlike the other commands.

**Comment by Staartvin** [comment]
You know that your time only updates every 5 minutes? Try lowering the interval setting in your Settings.yml.

**Comment by jokaacake** [comment]
Yes, I know that. When I said "too often" I didn't mean once in 30 seconds, it's like once in 15 minutes, my bad. 
Tried that and didn't work too 😞

**Comment by Staartvin** [comment]
That's strange. Do you have any exempt permissions? When Autorank is tracking your activity correctly, but not your time, that's weird.

**Comment by jokaacake** [comment]
That's it! You made me realize something saying "permissions". I'm using pex and some plugins don't give total permission to an OP user so I was using the permission "*" that makes me have all permissions.
By checking the AutoRank permissions again I found the permission "autorank.timeexclude" exists and that permission was blocking me from have time added to my total time, I was so dumb!
Anyway, I'm happy I finally figured it out. Many thanks for your help and patience and sorry if I bothered you or made you waste your time.
